### PR TITLE
Add completion to present and dismiss methods

### PR DESCRIPTION
**Pull request by ValeriyMaliuk**
**Description**:

**Related issue(s)**:

### CODE REVIEW

**Review by DmitryPashinskiy — APPROVED** [review]

File: ios/RIBs/Classes/Router.swift [hunk 1]
```diff
@@ -72,11 +72,12 @@ public protocol Routing: RouterScope {
...
     ///
     /// - Parameter router: `ViewableRouting` that should be presented
-    func present(_ router: ViewableRouting)
+    /// - Parameter completion: The block to execute after the presentation finishes.
+    func present(_ router: ViewableRouting, completion: (() -> Void)?)
 
     /// Dismisses current presented `ViewableRouting`
     /// - note: This method by defauls calls `present` of it's parent `Routing`
-    func dismiss()
+    func dismiss(completion: (() -> Void)?)
```
Comment (oboje) [inline ios/RIBs/Classes/Router.swift:80]: why we dont use Completable here as a result?
@Granfalloner fyi.

Comment (Granfalloner) [reply ios/RIBs/Classes/Router.swift:80]: Maybe `Single`, in case we opt in to Rx approach.

Comment (Granfalloner) [reply ios/RIBs/Classes/Router.swift:80]: @ValeriyMaliuk

Comment (ValeriyMaliuk) [reply ios/RIBs/Classes/Router.swift:80]: @oboje @Granfalloner We can. But should we? At first glance I don't see case when we will benefit from using Rx during presentation or dismissal. Usually they are done from `Router`s and calls from `Interactor` to `Router` is synchronous. Thus if there are no benefits and closure is simpler(no need to import Rx, no `DisposeBag` needed, syntax is cleaner) - closure may be a good choice here.

**Pull request merged**